src/scores/continuous/standard_impl.py

```diff
@@ -341,3 +341,57 @@ def multiplicative_bias(
     fcst, obs = broadcast_and_match_nan(fcst, obs)
     multi_bias = fcst.mean(dim=reduce_dims) / obs.mean(dim=reduce_dims)
     return multi_bias
+
+
+def additive_bias_percentage(
+    fcst: XarrayLike,
+    obs: XarrayLike,
+    *,
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    weights: Optional[XarrayLike] = None,
+) -> XarrayLike:
+    """
+    Calculates the percentage bias which is ratio of additive_bias and mean obs.
+
+    It is defined as
+
+    .. math::
+        \\text{Additive bias percentage} =\\frac{\\sum_{i=1}^{N}(x_i - y_i)}{\\sum_{i=1}^{N} y_i}
```

Comment:
mathjax looks good. I wonder if it would be easier again to read if there was a line break before "where x = ... "

Reply:
Updated.